www/src/components/forum/topic-link.tsx

```diff
@@ -0,0 +1,37 @@
+import Link from "next/link";
+import React from "react";
+import { usePostHog } from "posthog-js/react";
+
+export function TopicLink({
+  id,
+  title,
+  category,
+  children,
+}: {
+  id: number;
+  title: string;
+  category: {
+    id: number;
+    externalId: string;
+    name: string;
+  };
+  children: React.ReactNode;
+}) {
+  const posthog = usePostHog();
+
+  function handleLinkClick() {
+    posthog.capture("USER_ACCESSED_SUMMARY", {
```

Comment:
❌  this doesn't show they accessed, just that they clicked a link. IMO that event should rather fire when the page is loaded